tests/framework/PostProcessors/Validation/test_validation_gate_pcm.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="all">
+  <RunInfo>
+    <WorkingDir>PhysicsGuidedCoverageMapping</WorkingDir>
+    <Sequence>mcRunExp,mcRunApp,mcRunMsr,PCM</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <TestInfo>
+    <name>framework/PostProcessors/Validation/test_validation_gate_pcm</name>
+    <author>dhuang</author>
+    <created>2021-07-21</created>
+    <classesTested>PostProcessors.Validation</classesTested>
```

Comment:
update this info to reflect the PCM class

Reply:
Updated.